arboretum/kubernetes/README.md

```diff
@@ -17,7 +17,84 @@ to include the fetchers and checks from this library in your downstream project.
 
 ## Fetchers
 
-Fetchers coming soon...
+### Cluster Resource
+
+* Class: [KubeClusterResourceFetcher][fetch-cluster-resource]
+* Purpose: Write the resources of stand-alone kubernetes clusters to the evidence locker (for managed clusters, other cluster list fetcher and cluster resource fetcher can be used such as for [IBM Cloud](ibm-cloud-cluster-list-fetcher)).
+* Behavior: Read a cluster list from `raw/auditree/compliance_config.json` and fetch resources from  specified in the cluster list.
```

Comment:
I believe that the cluster list is read from the configuration not the evidence. Suggest:

```suggestion
* Behavior: Retrieve stand-alone Kubernetes cluster resource data for the provided list of clusters.  TTL is set to 1 day.
```